politeiad/backendv2/tstorebe/plugins/comments/cmds.go

```diff
@@ -560,9 +570,19 @@ func (p *commentsPlugin) cmdEdit(token []byte, payload string) (string, error) {
 		return "", err
 	}
 
+	// Ensure no extra data provided if not allowed
+	if !p.allowExtraData && (e.ExtraData != "" || e.ExtraDataHint != "") {
+		return "", backend.PluginError{
+			PluginID:     comments.PluginID,
+			ErrorCode:    uint32(comments.ErrorCodeExtraDataNotAllowed),
+			ErrorContext: "comment extra data is not allowed",
```

Comment:
Isn't this saying the same thing as the human readable `ErrorCodeExtraDataNotAllowed`?  Seems redundant.